plugins/services/src/js/containers/services/ServicesQuotaView.tsx

```diff
@@ -46,13 +46,14 @@ class ServicesQuotaView extends React.Component<ServicesQuotaViewProps, {}> {
       this.context.router.push("/services/overview/create");
     };
     const tabs = this.getTabs();
```

Comment:
We also need to update the tabs when in detail view so that `Services` takes you to the service detail for the group and not back to overview.

Reply:
Do we want this for this PR? I don't remember seeing this in the design.

Reply:
I'm not sure if its in the design, but I think that matches the expected UX IMO.

Reply:
Let's ask @izharbuendia .